### PR TITLE
Font Size Picker: Use a menuitemradio role and better labels.

### LINKED ISSUE #12371: Font size picker small accessibility improvements
I'd like to propose a couple small accessibility improvements for the font size picker:

screenshot 2018-11-27 at 20 00 05

Right now, there's no semantic indication of the currently selected font size:
- the toggle button could include this info in its aria-label
- the menu items should use an ARIA role `menuitemradio`, as this menu allows for a single choice 
- add an `aria-checked` attribute based on the selected item

### PR DESCRIPTION
**Pull request by afercia**
## Description

This PR seeks to introduce a few small improvements for the font size picker:

- adds the currently selected font size name to the toggle button `aria-label`
- uses an ARIA role `menuitemradio` on the menu items
- adds an `aria-checked` attribute to communicate which the selected item is
- removes the `aria-label` from the Reset button: the text "Reset" is enough
- removes `withInstanceId`: seems to me it wasn't used at all

Screenshots:

When focusing the toggle button, the currently selected font size is included in the button aria-label:

screenshot 2018-11-27 at 19 34 36

When navigating through the menu items, the selected state of the selected item is announced:

screenshot 2018-11-27 at 19 35 55

Fixes #12371

### CODE REVIEW

**Review by jorgefilipecosta — APPROVED** [review]
Everything looked correct in my tests I did not notice any regression 👍

File: packages/components/src/font-size-picker/index.js [hunk 1]
```diff
@@ -51,26 +51,34 @@ function FontSizePicker( {
...
-								</Button>
-							) ) }
+							{ map( fontSizes, ( { name, size, slug } ) => {
+								const isSelected = ( value === size || ( ! value && slug === 'normal' ) );
+
+								return (
+									<Button
+										key={ slug }
+										onClick={ () => onChange( slug === 'normal' ? undefined : size ) }
+										className={ 'is-font-' + slug }
```
Comment (tofumatt) [inline packages/components/src/font-size-picker/index.js]: Just a little thing, but we can use template strings here:
```suggestion
										className={ `is-font-${ slug }` }
```

File: packages/components/src/font-size-picker/index.js [hunk 2]
```diff
@@ -112,4 +119,4 @@ function FontSizePicker( {
 	);
 }
 
-export default withInstanceId( FontSizePicker );
```
Comment (tofumatt) [inline packages/components/src/font-size-picker/index.js]: I guess this isn't related, but good catch removing the unneeded HOC 👍

**Review by tofumatt — APPROVED** [review]
Looks good to me too, thanks! 👍

File: packages/components/src/font-size-picker/index.js (hunk 2 above)
Comment (afercia) [reply packages/components/src/font-size-picker/index.js]: As noted in the PR description 🙂

Comment (tofumatt) [reply packages/components/src/font-size-picker/index.js]: Oops, right. Sorry, my pre-weekend brain missed it 😉

**Pull request merged**